fe/src/pages/BufferPage.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
```

Comment:
- is this the `AuthMiddleware`?

Reply:
@escbooster12-dev  yes sir, Ill remove it